src/components/ContentWidgets/Accordion/AccordionEditor.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { Accordion } from './Accordion';
+
+export const AccordionEditor: React.FC<WidgetEditorProps> = ({
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    const emptySection = {
```

Comment:
this could be made outside of AccordionEditor, and you could have a specific type for this.

Reply:
I currently put this const object in a separate file `AccordionConstants.ts`. like 
```export const emptyPage = {
    title: "",
    panel: ""
}```
I'm not sure why and how to use it as a type. Do I declare an object like this?
```const page: emptyPage = {
    title: "",
    panel: ""
}```